azkaban-execserver/src/main/resources/log4j.properties

```diff
@@ -1,12 +1,12 @@
-log4j.rootLogger=INFO, Console
+log4j.rootLogger=INFO, ExecServer
 log4j.logger.azkaban.execapp=INFO, ExecServer
 
-log4j.appender.ExecServer=org.apache.log4j.RollingFileAppender
+log4j.appender.ExecServer=org.apache.log4j.DailyRollingFileAppender
```

Comment:
I believe that is case only when multiple JVMs are sending logs to same file.  In our case, it should be OK.